### PR TITLE
#issue #21 | Enhance: PageNotFound component with styling and icon

### PR DESCRIPTION
**Pull request by akshatshukla13**
1. Improve PageNotFound for better user experiance. 
2. Added sad face icon and backgroud. 
3. Now it has links to go home and see About page.
4. Introduced animations for icon and heading to make it dynamic.

### CODE REVIEW

**Comment by akshatshukla13** [comment]
light weight | used react icon for emoji animations

File: frontend-voting/src/components/PageNotFound.tsx [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+      <div className="relative z-10 flex flex-col justify-center items-center text-center">
+        <FaSadCry className="text-6xl mb-4 animate-bounce" />
+        <h1 className="text-8xl font-extrabold animate-pulse">404</h1>
+        <h2 className="text-3xl font-semibold mt-4">Oh no! Page Not Found</h2>
+        <p className="mt-2 text-lg text-gray-300">
+          The page you are looking for does not exist. But don’t worry, you can find your way back!
+        </p>
+        <Link 
+          to="/" 
+          className="mt-6 px-6 py-3 bg-blue-600 rounded-lg shadow-lg hover:bg-blue-700 transition duration-300 transform hover:scale-105"
```
Comment (skartik-sk) [inline frontend-voting/src/components/PageNotFound.tsx:19]: may be you can just change the color of the btn inorder to be in the same ui as that of home page.

**Comment by namansharma3007** [comment]
> light weight | used react icon for emoji animations

Please attach screenshots for POW

**Comment by akshatshukla13** [comment]
Screenshot (275)

**Comment by akshatshukla13** [comment]
The buttons that are visible are related to another issue related to outlets and routing. I'll create issue to address and resolve them.

**Pull request merged**